tests/sort/test_sort.f90

```diff
@@ -15,26 +16,26 @@ program test_sort
 
 b = [5, 4, 3, 2, 1]
 call sort(b)
-call assert(all(b == [1, 2, 3, 4, 5]))
+call assert(all(dabs(b - [1, 2, 3, 4, 5]) <= eps))
```

Comment:
Use `abs`, instead of `dabs`. Also I usually just use `< eps`, instead of `<= eps`, but that doesn't matter that much:
```suggestion
call assert(all(abs(b - [1, 2, 3, 4, 5]) < eps))
```